rmf_task/test/integration/test_backups.cpp

```diff
@@ -101,6 +143,8 @@ SCENARIO("Back up to file")
 
   // When the task is restored, it should resume where mock_active_task left off
   // and it should issue a phase snapshot to reflect that
+  mock_restored_task->start_next_phase(rmf_traffic::Time());
+  mock_restored_task->_active_phase->send_update();
```

Comment:
When a restoring a task, it should not be necessary to tell it to start the next phase. It should automatically begin the phase that the task left off from, and it should automatically issue a new update.